config.json.sample

```diff
@@ -0,0 +1,3 @@
+{
+    "featureManagerIsFullZoneProvisioningEnabled": false
+}
```

Comment:
@manatarms sorry to make you change again. Could you only add those 3 settings in the sample. 

Thanks